src/Networking.ino

```diff
@@ -1071,6 +1071,7 @@ bool downloadFile(const String& url, String file_save, const String& user, const
       size_t c = stream->readBytes(buff, std::min((size_t)len, sizeof(buff)));
 
       if (c > 0) {
+        timeout = millis() + 2000;
```

Comment:
Just curious, why updating this timeout?
And how does it differ to the internal timeout of the client object?

Reply:
my idea was to refresh timeout if stream is received even slowly

Reply:
It seems to work, so I've read :)